### ISSUE TITLE
Multiliveries tab is blank

### ISSUE THREAD

**Issue by TheUSFan** [Spice9/Geofs-Multiliveries#257]
First, I launch GeoFS, then I copy and paste the code in the console. I have been doing this for a long time ago, but today it didn't work. I mean, the "multiliveries" option was displayed in the game, but when I opened it, it was just a blank page, then it led me to Replit. What should I do? I do not know how to obtain the multiliveries anymore. Can you help me, please?

Thank you!

**Comment by Ariakim-Taiyo** [comment]
It should still be functional. I just rebooted the replit project hosting the multiliveries menu site just to be sure.
It may be an issue with your network. If you are on a school or similarly managed network, then check with an administrator to ensure replit is not blocked by a filter.
If the issue persists and you are on an unmanaged network, then try setting `debug = true` in the console when pasting the code. It is the first line of code.
Then try to use the addon as normal and report anything logged to the console by the plugin (not GeoFS).

**Comment by TheUSFan** [comment]
Thank you for the advice! Replit is functional for me, I'm going to find the issue, it is most probably mine.

**Comment by TheUSFan** [comment]
Just opened the game, it's working now. Thanks!

**Comment by Ariakim-Taiyo** [comment]
Alright, thanks for letting me know!
If you have any problems in the future, feel free to open another issue and I'll help as best I can.
